.github/contributing/docs_guide.md

```diff
@@ -0,0 +1,74 @@
+# Documentations Guide
+
+This guide aims to ensure consistency in the Bevy crates documentation, they should be seen as advice and not strict rules and should therefore be adapted on a case-by-case basis.
+
+## Prelude module's documentations
+
+If the crate has a prelude module, its documentation should be a single sentence with the name of the crate in camel_case and in code format (i.e. between backticks `` `bevy_XXX` ``).
+
+```rust
+/// The `bevy_XXX` prelude.
+```
+
+Example for `bevy_animation`
+
+```rust
+/// The `bevy_animation` prelude.
+pub mod prelude {
+    /* ... */
+}
+```
```

Comment:
I would prefer to not document those at all.

Adding "the `crate_name` prelude" doesn't bring any value. Something useful would be the information of what to expect from the prelude, what a user could use it for and which use cases are covered with it, but I think that would be too much noise/harder to maintain, and not the place people would look for that anyway.